web-console/future-openshift/diagnostics/diagnostics.md

```diff
@@ -0,0 +1,55 @@
+# Generating Diagnostic Report
+
+Users can download a cluster diagnostic report based on must-gather tool for upload to Red Hat Support or their own diagnostics.
+
+## Accessing 'Generate Report' actions
+
+There are several ways the user can access the `Generate (diagnostics) report` action.
+
+### Via the masthead help menu
+![Masthead help click](img/1-1a-clusterdash-startdownload.png)
+![Masthead help menu](img/1-1b-clusterdash-startdownload.png)
+
+### Via the cluster dashboard status card kebab menu
```

Comment:
just a note, in the 4.3 timeframe the "filter alerts" is not going to be present in the dashboard so it can not be integrated here. We can keep the design for 4.4 though.

Reply:
Thanks, removed the filter from this 4.3 design.